config/3.3/ai-manager/values.large.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Maybe we can drop the `values.<profile>.yaml` files as it does not add too much value while introducing additional effort to maintain consistency due to most of the contents are duplicated.